src/components/ImportButton.vue

```diff
@@ -1,22 +1,49 @@
 <script setup>
-import { useRouter } from 'vue-router';
-import { inject } from 'vue';
-
-const router = useRouter();
-const selectedImages = inject('selectedImages');
-
-const goToDataSession = () => {
-  if (selectedImages.value && selectedImages.value.length > 0) {
-    router.push({ 
-      name: 'DataSession', 
-      params: { selectedImages: JSON.stringify(selectedImages.value) }
-    });
-  } else {
-    alert("No images selected");
+import { useRouter } from 'vue-router'
+import { useStore } from 'vuex'
+
+const router = useRouter()
+const store = useStore()
+
+const importImages = async () => {
+  try {
+    const selectedImages = store.state.selectedImages
+    const inputData = selectedImages.map(image => ({
+      'source': image.image,
```

Comment:
the source key being `image.image` seems a bit confusing, but maybe that is tangential to this PR. I would think a property on the image like `source` would make more sense? I know its all placeholder data for now though...

Reply:
oh yeah that totally makes sense. The mock data has the images saved as images. I'll change that after break if that's okay : )